src/components/memberPage/members.json

```diff
@@ -386,6 +386,17 @@
         "Front-end design",
         "Spatial data management"
       ]
+    },
+    {
+      "name": "Rong Chao",
+      "img": "roychao.jpeg",
+      "mail": "f14071075@gs.ncku.edu.tw",
+      "link": {
+        "github": "https://github.com/RoyChao19477"
```

Comment:
You need to set `linkedin` and `homepage` to `null` or the page can not be rendered correctly.
```javascript
"link": {
    "github": "https://github.com/coliver0221",
    "linkedin": null,
    "homepage": null
},
```

Reply:
Thanks ! I've updated the `linkedin` and `homepage` to `null`. Please let me know if there is anything still need to be revised.